pkg/client/client.go

```diff
@@ -103,12 +101,23 @@ func Initialize(
 
 	for _, keepAddress := range keepsRegistry.GetKeepsAddresses() {
 		go func(keepAddress common.Address) {
-			isActive, err := ethereumChain.IsActive(keepAddress)
+			keep, err := ethereumChain.GetKeepWithID(keepAddress)
+			if err != nil {
+				logger.Errorf(
+					"failed to verify if keep [%s] is still active: [%v]; "+
+						"subscriptions for keep signing and closing events are skipped",
+					keep.ID(),
+					err,
+				)
```

Comment:
We use the same error message as for the `keep.IsActive()` check below. Can we differentiate them a little so that the node operator can be sure which call failed?

Reply:
Done.